tasks/examples.py

```diff
@@ -31,7 +31,7 @@ def build(ctx, clean=False):
         [
             "cmake",
             "-GNinja",
-            "-DCMAKE_BUILD_TYPE=Release",
+            "-DCMAKE_BUILD_TYPE=Debug",
```

Comment:
Do we need to change the release type here? Ideally the examples would be treating Faabric as an external project, hence using the `Release` build (not a big deal though if there's a reason to change it).